runtime/src/server/middleware/get_page_handler.ts

```diff
@@ -294,6 +294,9 @@ export function get_page_handler(
 				script += `</script><script src="${main}">`;
 			}
 
+			// users can set a CSP nonce using res.locals.nonce
+			const nonce_attr = (res.locals && res.locals.nonce) ? ` nonce="${res.locals.nonce}"` : '';
```

Comment:
Would you be able to rebase this PR? This line was moved higher up in the file in https://github.com/sveltejs/sapper/pull/1395.

Reply:
I've rebased.